pkg/apiserver/api_installer.go

```diff
@@ -567,6 +571,24 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 			}
 			addParams(route, action.Params)
 			ws.Route(route)
+		case "DELETECOLLECTION":
+			doc := "delete collection of " + kind
+			if hasSubresource {
+				doc = "delete collection of " + subresource + " of a " + kind
```

Comment:
not obvious to me, but this doesn't appear to be used.  where would this be useful?  am i missing an obvious kind?